doc/backends.rst

```diff
@@ -322,8 +322,8 @@ from the previous section):
 Compile the Nim code to JavaScript with ``nim js -o:fib.js fib.nim`` and
 open ``mhost.html`` in a browser. If the browser supports javascript, you
 should see an alert box displaying the text ``Fib for 9 is 34``. As mentioned
-earlier, JavaScript doesn't require an initialisation call to ``NimMain`` or
-similar function and you can call the exported Nim proc directly.
+earlier, JavaScript doesn't require an initialization call to ``NimMain`` or
```

Comment:
not sure about this one (and other similar cases) - `initialisation` is valid British English. Maybe ask @narimiran @Araq ?

Reply:
You have a point here - I have "American"-ized some words. If there is any issue with this change, I can revert it.